### PR TITLE
task-10: Làm trang home

### PR DESCRIPTION
**Pull request by chaukhoa97**
https://edu-redmine.sun-asterisk.vn/issues/38910

### CODE REVIEW

**Comment by chaukhoa97** [comment]
ready

File: src/sass/base/_typography.scss [hunk 1]
```diff
@@ -21,6 +21,10 @@ label,
 option,
 select,
 .form-select,
+<<<<<<< HEAD
```
Comment (manhnt-0960) [inline src/sass/base/_typography.scss]: .

File: src/sass/pages/_product-detail.scss [hunk 2]
```diff
@@ -48,13 +48,17 @@
   }
 
   &__link {
+<<<<<<< HEAD
+    @include scroll-enable;
+=======
```
Comment (manhnt-0960) [inline src/sass/pages/_product-detail.scss]: .

**Comment by chaukhoa97** [comment]
ready

**Pull request merged**